gradlew

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/usr/bin/env sh
```

Comment:
I think this file could probably be reverted too?

Reply:
Yep think this is the work of `gradle wrapper` command

Reply:
Sorry, I meant I think this whole file should be reverted. We're not bumping gradle or anything as part of this change, so I don't think the gradlew script needs to change at all.